src/components/Proposals/InfoBoxes.js

```diff
@@ -0,0 +1,187 @@
+                the tokens used to vote on a proposal must vote “Yes” for it to
+                pass.
+              </Help>
+            </>
+          }
+          css={`
+            height: 100%;
+          `}
+          padding={(compactMode ? 2 : 3) * GU}
+        >
+          <SummaryWithPercentages
+            size={support}
+            requiredSize={parseFloat(setting.formattedSupportRequiredPct)}
+          />
+        </Box>
+      </div>
+      <div>
+        <Box
+          heading={
+            <>
+              Minimum Approval %
+              <Help hint="What is Minimum Approval?">
+                <strong>Minimum Approval</strong> is the percentage of the total
+                token supply that is required to vote “Yes” on a proposal before
+                it can be approved. For example, if the “Minimum Approval” is
+                set to 20%, then more than 20% of the outstanding token supply
+                must vote “Yes” on a proposal for it to pass.
+              </Help>
+            </>
+          }
+          css={`
+            height: 100%;
+          `}
+          padding={(compactMode ? 2 : 3) * GU}
+        >
+          <SummaryWithPercentages
+            size={quorumProgress}
+            requiredSize={parseFloat(
+              setting.formattedMinimumAcceptanceQuorumPct
+            )}
+          />
+        </Box>
+      </div>
+    </div>
+  )
+}
+
+function SummaryWithPercentages({ size, requiredSize }) {
+  const theme = useTheme()
+
+  return (
+    <>
+      <div
+        css={`
+          ${textStyle('body2')};
+        `}
+      >
+        {round(size * 100, 2)}%{' '}
+        <span
+          css={`
+            color: ${theme.surfaceContentSecondary};
+          `}
+        >
+          (&gt;{requiredSize}% needed)
+        </span>
+      </div>
+      <SummaryBar
+        positiveSize={size}
+        requiredSize={requiredSize / 100}
+        css={`
+          margin-top: ${2 * GU}px;
+        `}
+      />
+    </>
+  )
+}
+
+function Status({ vote }) {
+  const theme = useTheme()
+  // TODO: get if vote is open
+  const { endDate, executionDate, open } = vote
+
+  if (open) {
+    return (
+      <>
+        <div
+          css={`
+            ${textStyle('body2')};
+            color: ${theme.surfaceContentSecondary};
+            margin-bottom: ${1 * GU}px;
+          `}
+        >
+          Time remaining
+        </div>
+        <Timer end={toMs(endDate)} maxUnits={4} />
+      </>
+    )
+  }
+
+  return (
+    <>
+      <div
+        css={`
+          margin-top: ${1 * GU}px;
+          display: inline-grid;
+          grid-auto-flow: column;
+          grid-gap: ${1 * GU}px;
+          align-items: center;
+          color: ${theme.surfaceContentSecondary};
+          ${textStyle('body2')};
+        `}
+      >
+        <IconTime size="small" />{' '}
+        {dateFormat(toMs(executionDate || endDate), 'standard')}
+      </div>
+    </>
+  )
+}
+
+InfoBoxes.propTypes = {
```

Comment:
We'll be moving these out later, but a nitpick here is that we normally place the propTypes right after the component it's declaring types for.